resources/backend/sql_transactions.md

```diff
@@ -0,0 +1,9 @@
+## What is a SQL transaction?
+
+A SQL transaction is a way to use commands in the database, but not have them actually affect a main database -- sort of a testing case.
+
+## How to use a SQL Transaction
+
+To create the testing database, use `transaction = db.Begin()`  
+To execute commands, continue to use `db.Execute('<Command>')`  
+To commit the database, use `transaction.Commit()`
```

Comment:
You should also include how to use Rollback().

And give some sample code of how to use this:

```
tx := db.Begin()
err := db.Exec(Command to do actionA)
if err != nil { // Command actionA failed, rollback transaction
   tx.Rollback()
   return err
}
err := db.Exec(Command to do actionB)
if err != nil { // Command actionB failed, rollback transaction
   tx.Rollback()
   return err
}

// Everything is a success, fully commit db transaction!
tx.Commit()
```